### ISSUE TITLE
Update Docs

### ISSUE THREAD

**Issue by CallumAdgt** [privacysandbox/aggregation-service#45]
The sample provided [here](https://github.com/privacysandbox/aggregation-service/blob/main/docs/collecting.md#aggregatable-report-sample) is using an out of date shared_info which also doesn't contain a version.

Better to use the one from the sampledata dir - here is the plaintext 

```
"{\"api\":\"attribution-reporting\",\"version\":\"0.1\",\"scheduled_report_time\":1698872400.000000000,\"reporting_origin\":\"http://adtech.localhost:3000\",\"source_registration_time\":1698796800.000000000,\"attribution_destination\":\"dest.com\",\"debug_mode\":\"enabled\",\"report_id\":\"b360383a-108d-4ae3-96bd-aecde1c3c30b\"}"
```

Which has an allowed version, an actual 'api' key and also has attribution_destination moved within shared_info.

**Comment by maybellineboon** [comment]
Hi @CallumAdgt ,

Thanks for reaching out. We are working on updating the [sample](https://github.com/privacysandbox/aggregation-service/blob/main/docs/collecting.md#aggregatable-report-sample). We'll update you once this is available.

For the meantime, if you are using AWS and want to test Aggregation Service out, we have a [codelab](https://developers.google.com/privacy-sandbox/codelab/aggregation-service-aws#0) that will allow you to create an aggregatable report to test with Aggregation Service. This should be available in Section 4.1 of the codelab.

**Comment by maybellineboon** [comment]
Closing as we're scheduling to publish the update. Workaround is as provided.
